crates/bin/pd/src/main.rs

```diff
@@ -341,6 +346,9 @@ async fn main() -> anyhow::Result<()> {
                 .add_service(tonic_web::enable(SpecificQueryServiceServer::new(
                     info.clone(),
                 )))
+                .add_service(tonic_web::enable(ClientQueryServer::new(ibc.clone())))
+                .add_service(tonic_web::enable(ChannelQueryServer::new(ibc.clone())))
+                .add_service(tonic_web::enable(ConnectionQueryServer::new(ibc.clone())))
```

Comment:
Non-blocking: going forward, we want to split up the RPCs and move them into the component crates. When we do that, we'll have LOTS of different services to stitch together, and we should figure out how to cleanly add them all. Maybe `tower::Layer`s would be helpful there.